File: app/src/main/java/remix/myplayer/helper/MusicEventCallback.java
package remix.myplayer.helper;

import remix.myplayer.service.MusicService;

public interface MusicEventCallback {
    void onMediaStoreChanged();
    void onPermissionChanged(boolean has);
    void onPlayListChanged();
    void onServiceConnected(MusicService service);
    void onMetaChanged();
    void onPlayStateChange();
    void onServiceDisConnected();
}

File: app/src/main/java/remix/myplayer/helper/MusicEventCallback.kt
package remix.myplayer.helper

import remix.myplayer.bean.mp3.Song
import remix.myplayer.service.MusicService

interface MusicEventCallback {
  fun onMediaStoreChanged()

  fun onPermissionChanged(has: Boolean)

  fun onPlayListChanged(name: String)

  fun onServiceConnected(service: MusicService)

  fun onMetaChanged()

  fun onPlayStateChange()

  fun onServiceDisConnected()

  fun onTagChanged(oldSong: Song, newSong: Song)
}
